installer/dbscript/dbscript-2.sql

```diff
@@ -3875,3 +3878,8 @@ VALUES (1, 1, 1, 1, 0, @apiv2_leave_holiday_data_group_id, @admin_role_id),
        (1, 0, 0, 0, 0, @apiv2_leave_holiday_data_group_id, @ess_role_id),
        (1, 0, 1, 0, 0, @apiv2_leave_workweek_data_group_id, @admin_role_id),
        (1, 0, 0, 0, 0, @apiv2_leave_workweek_data_group_id, @ess_role_id),
+       (1, 0, 0, 0, 1, @apiv2_pim_report_to_allowed_employees_data_group_id, @ess_role_id),
```

Comment:
better if you can move new rules under PIM related insert query above, since you added them under leave related insert query